shuup/front/template_helpers/general.py

```diff
@@ -24,6 +24,13 @@
 from shuup.front.utils.views import cache_product_things
 from shuup.utils.mptt import get_cached_trees
 from shuup.utils.translation import cache_translations_for_tree
+from shuup.front.apps.auth.forms import EmailAuthenticationForm
+
+
+@contextfunction
+def get_login_fields(context, request):
+    form = EmailAuthenticationForm(request=request)
```

Comment:
This shouldn't be hardcoded.

SHUUP_LOGIN_VIEW=“shuup.front.apps.auth.views:LoginView” Put this under `settings.py` under `shuup.front`. Then in templatetag `from shuup.utils.importing import cached_load` `login_class = cached_load("SHUUP_LOGIN_VIEW")`. Finally take the form from the view instead.